.github/ISSUE_TEMPLATE/documentation_report.md

```diff
@@ -15,7 +15,7 @@ about: Ask us about docs
 - Documentation Report
 
 ##### COMPONENT NAME
-<!--- Write the name of the rst file, module, plugin, task or feature below -->
```

Comment:
We added the RST file on purpose not so long ago based on feedback.
Are we removing this on purpose ?

Reply:
Oh, that should be URL of page, or URL of where you'd expect this to be documented.

Reply:
Honestly, I think we should be asking people to edit the documentation on the website, rather than open an issue like this.

Reply:
re added rst